Automattic/jetpack
# Social Menu: Syncing changes from WordPress.com to Jetpack

#### Changes proposed in this Pull Request:

* This PR is the first part in de-fusioning Social Menus: https://github.com/Automattic/jetpack/issues/25234
* In this PR, changes that exist in the WordPress.com versions of the files are added to the Jetpack file versions.

#### Other information:


#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* To test, add the Twenty Twenty theme to a Jetpack-connected site and in the `functions.php` file (or via a code snippets plugin) add the following: `add_theme_support( 'jetpack-social-menu', 'svg' );`. This is to make sure we test the `icon-functions.php` file as well.
* Make sure to add a social menu via `wp-admin/nav-menus.php`.
* The icons should display as expected without any errors.


~As a general note, I haven't had any luck in getting `has_nav_menu('jetpack-social-menu'))` to return true, even if a jetpack-social-menu is set. These calls existed before (in `icon-functions.php` and `social-menu.php`) - so not related to this PR.  Everything appears to work fine, but that makes me a little uneasy about the existing code.~ (solved below)
